cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,76 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SettingsPageObject from '../support/pages/settings.PageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-describe('Settings page', () => {
-  before(() => {
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  });
+describe('Settings page', () => {
+  let newData;
+  let user;
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateNewData').then(generateNewData => {
+      newData = generateNewData;
+    });
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.visit();
+    settingsPage.typeNewUsername(newData.username);
+    settingsPage.clickUpdateSettingsBtn();
+    settingsPage.assertMessageAboutUpdatingData();
+    homePage.assetrNewUsername(newData.username);
```

Comment:
Typo in the `homePage.assetrNewUsername()` method. It should be `homePage.assertNewUsername()`.